### ISSUE TITLE
JAWS says 'clickable' on non-clickable elements

### ISSUE THREAD

**Issue by LilyL0u** [bbc/simorgh#9545]
Found when testing the recommendation component, and then also found when checking the related content component.

To reproduce:
1. Use JAWS
2. Navigate to the image in a component (with arrow keys) with a list of onward journey links e.g https://www.bbc.com/mundo/noticias-58287409 for recommendations and https://www.bbc.com/afrique/monde-58899290 for related content. Hear that JAWS reads 'clickable' after the alt-text and saying 'graphic'.
3. Use the enter key to click the image. See that you are not navigated to anything. 
4. Tab to the link text which is the next focusable element. When you press enter, you will be taken to the linked page.

It is expected that the image should not be clickable with a screen reader as this is not within a HTML link element.

Also the title of the section 'You may also be interested in', also says 'clickable' when it is not, and should not be.

**Comment by LilyL0u** [comment]
This is not happening on the new version of JAWS. I was using v18. v2021 does not have the bug.
